crates/project/src/git.rs

```diff
@@ -1252,7 +1252,7 @@ impl GitStore {
 
         repository_handle
             .update(&mut cx, |repository_handle, cx| {
-                repository_handle.commit(message, name.zip(email), cx)
+                repository_handle.commit(message, name.zip(email), false, cx)
```

Comment:
I'm not sure where/how this code is executed exactly, but I have no reference here to the Git panel or modal as far as I'm aware. Defaulting to false seems sensible, but if I'm missing something let me know.